src/components/icon/icon.scss

```diff
@@ -249,15 +249,17 @@
 }
 
 .icon-vr-stereo-full {
-  background-image: icon(vrStereoFull, $brand-color);
+  --playkit-icon-data-url: #{icon(vrStereoFull, $brand-color)};
```

Comment:
where do the default values for the variables come from ?

Reply:
sass icon function